storage/src/main/java/org/apache/uniffle/storage/handler/api/ShuffleDeleteHandler.java

```diff
@@ -25,4 +25,11 @@ public interface ShuffleDeleteHandler {
    * @param appId ApplicationId for delete
    */
   void delete(String[] storageBasePaths, String appId, String user);
+
+  /**
+   * Delete shuffle data with appId
+   *
+   * @param appId ApplicationId for delete
+   */
+  void delete(String[] storageBasePaths, String appId);
```

Comment:
Why do we need this interface? Could we reuse `delete(String[] storageBasePaths, String appId, String user)`?